### ISSUE TITLE
Loss not converging of IP adapter training on  finetuned sd 1.5 model

### ISSUE THREAD

**Issue by avinashthakur1590** [tencent-ailab/IP-Adapter#353]
while training a ipadapter model over my finetuned stable diffusion 1.5 model, i am observing training loss not converging properly, i have not altered the default learning rates or the optimizers they are same as in repo. or the reference image is not able to condition well the ip adapter and i need to modify the dataset
